<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StimulationCreator</class>
 <widget class="QWidget" name="StimulationCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stimulation editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QWidget" name="editor" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="randomise">
            <property name="text">
             <string>Randomise</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="nSteps">
            <property name="suffix">
             <string> steps</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QTableWidget" name="steps">
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>24</number>
            </attribute>
            <column>
             <property name="text">
              <string>Ramp</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Voltage</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="duration">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="maximum">
             <double>5000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="baseV">
            <property name="suffix">
             <string> mV</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QDoubleSpinBox" name="tObsBegin">
            <property name="prefix">
             <string>Observe from </string>
            </property>
            <property name="suffix">
             <string> ms</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="tObsEnd">
            <property name="prefix">
             <string>to </string>
            </property>
            <property name="suffix">
             <string> ms</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="copy">
            <property name="text">
             <string>Copy</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="paste">
            <property name="text">
             <string>Paste</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0" colspan="3">
           <widget class="QTableWidget" name="stimulations">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>50</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>24</number>
            </attribute>
            <column/>
            <column/>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Stimulations</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="save">
            <property name="text">
             <string>Save set</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="nStim">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QComboBox" name="sources"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCustomPlot" name="plot" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sources</tabstop>
  <tabstop>nStim</tabstop>
  <tabstop>save</tabstop>
  <tabstop>stimulations</tabstop>
  <tabstop>randomise</tabstop>
  <tabstop>copy</tabstop>
  <tabstop>paste</tabstop>
  <tabstop>nSteps</tabstop>
  <tabstop>duration</tabstop>
  <tabstop>baseV</tabstop>
  <tabstop>tObsBegin</tabstop>
  <tabstop>tObsEnd</tabstop>
  <tabstop>steps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
